docs/options.md

```diff
@@ -0,0 +1,10 @@
+# Driver Options
+There are couple driver options that can be passed as arguments when starting driver container.
+
+| Option argument             | value sample                                      | default                                             | Description         |
+|-----------------------------|---------------------------------------------------|-----------------------------------------------------|---------------------|
```

Comment:
Use a consistent format in the Description column. Complete sentences with capitalization and periods at the end. Or not. Don't mix and match.